src/calculateRentalCost.js

```diff
@@ -5,6 +5,20 @@
  */
 function calculateRentalCost(days) {
   // write code here
+
+  if (days === 7) {
+    return 230;
+  }
+
+  if (days === 6) {
+    return 220;
+  }
+
+  if (days === 3) {
+    return 100;
+  }
+
+  return 80;
```

Comment:
The current implementation uses hardcoded values for specific days (3, 6, and 7), which might not be flexible enough if the task requires handling other day ranges or a general formula. Verify the task requirements to ensure all necessary scenarios are covered.